### DISCUSSION TITLE
Set X and Y values on scrollTo(edge)

### DISCUSSION THREAD

**Discussion by iMaupin** [wix/Detox#3423] [Q&A]
The [documentation](https://github.com/wix/Detox/blob/master/docs/APIRef.ActionsOnElement.md#scrolltoedge) seems to indicate that this is not available. Could the x and y values on this method be exposed like they are on other, incremental scroll methods?

**Comment by asafkorem** [comment]
Hi @iMaupin,
Please check the regular `scroll` method: [here](https://github.com/wix/Detox/blob/master/docs/APIRef.ActionsOnElement.md#scrolloffset-direction-startpositionx-startpositiony).

Isn't that enough for your needs?

For example:
```js
await element(by.id('scrollView')).scroll(50, 'up');
await element(by.id('scrollView')).scroll(50, 'down');
```

**Reply by iMaupin** [reply]
@asafkorem 

Thanks for the reply. When I asked this, I am pretty sure that I was hoping to be able to configure the scrollTo(Edge) method to allow for me to not have to set a specific scroll amount, while—at the same time—configuring where on the screen the tap begins.

**Reply by asafkorem** [reply]
Oh OK, sorry, I got confused and now I get it. You want to define the starting position.

Currently this is indeed not available, but we will definitely consider adding these params. Is it still relevant to you?
Also, I would appreciate it if you could explain a little about your use case, if this feature is still relevant for you.

**Reply by iMaupin** [reply]
Yeah, it's definitely still relevant. Basically, I am trying to avoid elements on the screen getting tapped unintentionally when I use the scrollTo('edge') methods. Being able to set the x or y coordinate as you can on scroll() should mitigate this issue.

**Reply by asafkorem** [reply]
Oh, I see..
I guess that it happens only in Android..? or you experience this on iOS as well?

**Comment by asafkorem** [comment]
@iMaupin, thanks for bringing this feature request to our attention, I've opened a new feature idea issue for that: https://github.com/wix/Detox/issues/3512

**Reply by iMaupin** [reply]
@asafkorem funny enough, the issue doesn't happen on my android test devices but does happen on the iOS simulator (a TextInput gets tapped as a ScrollView executes scrollTo('bottom')).

**Reply by asafkorem** [reply]
I asked because we saw a similar issue with Android, lol 😅
